querydsl-elasticsearch2/pom.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.querydsl.contrib</groupId>
+    <artifactId>querydsl-contrib</artifactId>
+    <version>4.0.9.BUILD-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>querydsl-elasticsearch2</artifactId>
+  <name>Querydsl - Elasticsearch support 2.x</name>
+  <description>Elasticsearch support for Querydsl</description>
+  <packaging>jar</packaging>
+  <url>${project.homepage}</url>
+
+  <scm>
+    <connection>${project.checkout}</connection>
+    <developerConnection>${project.checkout}</developerConnection>
+    <url>${project.githubpage}</url>
+  </scm>
+
+  <properties>
+    <elasticsearch.version>2.2.2</elasticsearch.version>
+    <jackson.version>2.6.5</jackson.version>
+    <osgi.import.package>
+      org.elasticsearch.*;version="0.0.0",
+      com.fasterxml.jackson.core.*;version="0.0.0",
+      ${osgi.import.package.root}
+    </osgi.import.package>
+  </properties>
+
+  <dependencies>
+
+    <!-- Elasticsearch -->
+    <dependency>
+      <groupId>org.elasticsearch</groupId>
+      <artifactId>elasticsearch</artifactId>
+      <version>${elasticsearch.version}</version>
+    </dependency>
+
+    <!-- Jackson JSON Mapper -->
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>${jackson.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>${jackson.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.querydsl</groupId>
+      <artifactId>querydsl-apt</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.springsource.bundlor</groupId>
+        <artifactId>com.springsource.bundlor.maven</artifactId>
```

Comment:
`<version />` not present.
I notice that it's not there on the original elasticsearch module. Thoughts @timowest ?